CI/appveyor.after_success.ps1

```diff
@@ -8,9 +8,10 @@ windeployqt.exe --release mudlet.exe
 
 Remove-Item * -include *.cpp, *.o
 
-$public_test_build = if ($Env:MUDLET_VERSION_BUILD) { $public_test_build } else { $FALSE }
+$public_test_build = if ($Env:MUDLET_VERSION_BUILD) { $Env:MUDLET_VERSION_BUILD.StartsWith('-public-test-build') } else { $FALSE }
```

Comment:
To me, the initial `$` on the variable name on the **left** side of the assignment operator seems suspicious - but this isn't a `bash` shell script but a `powershell` one so I know very *little* and am not in a position to give an informed opinion. OTOH this is used in several places in this file so I guess it is valid.

Reply:
Indeed it is!